operators/tokenizer/bpe_kernels.cc

```diff
@@ -133,6 +133,14 @@ std::vector<int64_t> KernelBpeTokenizer::Tokenize(ustring& input,
   std::vector<int64_t> res;
   std::list<std::pair<uint32_t, uint32_t>> byte_list;
 
+  // HF implements a cache for BPE:
+  // https://github.com/huggingface/transformers/blob/6f316016877197014193b9463b2fd39fa8f0c8e4/src/transformers/models/gpt2/tokenization_gpt2.py#L216C6-L216C6
+
+  // We use a LRU cache algorithm for the same in C++ in order to save compute.
+
+  // Current cache capacity is set to a relatively small 500 in order to support mobile platforms.
+  LRUCache bpe_cache = LRUCache(500);
```

Comment:
can it be used for next Compute call?

Reply:
no - if we move it outside to use it for the next Compute call it throws errors because the type of BPE tokenizer (GPT2/CLIP/Roberta) is different on different calls and the tests fail because ids/offsets are wrong.